tests/functional/base_test.go

```diff
@@ -53,7 +53,13 @@ func GetNonDefaultWatcherSpec() map[string]interface{} {
 		"tls": map[string]interface{}{
 			"caBundleSecretName": "combined-ca-bundle",
 		},
-		"prometheusSecret": "custom-prometheus-config",
+		"prometheusSecret":         "custom-prometheus-config",
+		"applierContainerImageURL": "fake-Applier-Container-URL",
+		"applierServiceTemplate": map[string]interface{}{
+			"replicas":            1,
+			"nodeSelector":        map[string]string{"foo": "bar"},
+			"customServiceConfig": "# Service config 2",
```

Comment:
maybe nicer to use " # Service config applier" so it is easier for the humans to follow 
(I'll be adding the same for the decisionengine https://github.com/openstack-k8s-operators/watcher-operator/pull/65/

Reply:
Done, thanks!